app/views/layouts/_header.html.erb

```diff
@@ -1,7 +1,7 @@
 <header>
   <div class="container">
     <nav class="navbar navbar-expand-lg navbar-dark">
-      <%= link_to "NVStory", root_path, class: "navbar-brand" %>
+      <%= link_to image_tag("logo_web.jpeg", alt: "Story logo", height: "50", width: "100"), root_path, class: "navbar-brand" %>
```

Comment:
cái này dùng css đc ko

Reply:
dạ đc ạ . Tai cái ảnh em kiếm trên mạng nó to em cắt thô sơ nên phải custom bằng tay :v